src/tagging/components/tagModifier.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TagSelector from './tagSelector';
+import ValueSelector from './valueSelector';
+
+const TagModifier = (props) => {
+  const tagValues = props.tags[props.selectedTagCategory] || [];
+  const tagCategories = Object.keys(props.tags) || [];
+  return (
+    <div className="col-lg-6">
+      <div className="row">
```

Comment:
please use col and row components from patternfly-react

Reply:
Done